firstrun/style.css

```diff
@@ -0,0 +1 @@
-textarea:focus{border-color:#00a4dc}.emby-select-withcolor{color:inherit;background:#292929;border:.07em solid #292929}.emby-select-withcolor>option{color:inherit;background:#222}.emby-select-withcolor:focus{border-color:#00a4dc!important}.emby-select-tv-withcolor:focus{background-color:#00a4dc!important;color:#fff!important}.emby-checkbox:checked+span+span+.checkboxOutline{border-color:#00a4dc}.emby-checkbox:focus+span+.emby-checkbox-focushelper{background-color:rgba(0,164,220,.26)}.emby-checkbox:checked+span+span+.checkboxOutline,.itemProgressBarForeground{background-color:#00a4dc}.itemProgressBarForeground-recording{background-color:#cb272a}.countIndicator,.fullSyncIndicator,.playedIndicator{background:#00a4dc}.fullSyncIndicator{color:#fff}.mainDrawer{background-color:#101010}.navMenuOption:hover{background:#252528}.navMenuOption-selected{background:#00a4dc!important;color:#fff}.emby-button-focusscale:focus{background:#00a4dc;color:#fff}.emby-tab-button{color:#999;color:rgba(255,255,255,.4)}.emby-tab-button-active{color:#fff}.emby-tab-button-active.emby-button-tv{color:#999;color:rgba(255,255,255,.4)}.emby-tab-button.emby-button-tv:focus{color:#fff;background:0}.channelPrograms,.guide-channelHeaderCell,.programCell{border-color:rgba(255,255,255,.05)}.programCell-sports{background:#3949ab!important}.programCell-movie{background:#5e35b1!important}.programCell-kids{background:#039be5!important}.programCell-news{background:#43a047!important}.programCell-active{background:#1e1e1e!important}.guide-channelHeaderCell:focus,.programCell:focus{background-color:#00a4dc!important;color:#fff!important}.guide-programTextIcon{color:#1e1e1e;background:#555}.guide-headerTimeslots{color:inherit}.guide-date-tab-button{color:#555;color:rgba(255,255,255,.3)}.guide-date-tab-button.emby-tab-button-active,.guide-date-tab-button:focus{color:#00a4dc}.guide-date-tab-button.emby-button-tv:focus{background-color:#00a4dc;color:#fff}.itemBackdropFader{background:-webkit-gradient(linear,left top,left bottom,from(rgba(0,0,0,0)),to(#101010));background:-webkit-linear-gradient(rgba(0,0,0,0),#101010);background:-o-linear-gradient(rgba(0,0,0,0),#101010);background:linear-gradient(rgba(0,0,0,0),#101010)}.infoBanner{color:#ddd;background:#111;padding:1em;-webkit-border-radius:.25em;border-radius:.25em}.ratingbutton-icon-withrating{color:#c33}.downloadbutton-icon-complete,.downloadbutton-icon-on{color:#4285f4}.playstatebutton-icon-played{color:#c33}.repeatButton-active{color:#4285f4}.card:focus .card-focuscontent{border-color:#00a4dc}.layout-desktop::-webkit-scrollbar{width:1em;height:1em}::-webkit-scrollbar-track{-webkit-box-shadow:inset 0 0 6px rgba(0,0,0,.3)}::-webkit-scrollbar-track-piece{background-color:#3b3b3b}::-webkit-scrollbar-thumb:horizontal,::-webkit-scrollbar-thumb:vertical{-webkit-border-radius:2px;background:center no-repeat #888}.timeslotHeaders-desktop::-webkit-scrollbar{height:.7em}.manualLoginForm{margin:0 auto}.manualLoginForm>h1{margin-top:1em;text-align:left}
```

Comment:
not sure this should be minimized

Reply:
@skaro13 thoughts?

Reply:
I agree since any other css for this project seems to be unminified.